lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -52,6 +52,7 @@ def commands
       "machine_id" => "Get the MSF ID of the machine attached to the session",
       "quit"       => "Terminate the meterpreter session",
       "resource"   => "Run the commands stored in a file",
+      "uuid"       => "Get the UUID for the current session",
```

Comment:
UUID is basically part of everything now (well, it's coming in Java and Android soon thanks to Tim), so I put it up in the main menu.